vignettes/tidyverse_translation.Rmd

```diff
@@ -84,10 +92,6 @@ This vignette is largely inspired from `{dplyr}`'s [Getting started vignette](ht
 </div>
 
 ```{r, eval = evaluate_chunk}
-library(dplyr)
-library(tidyr)
-library(datawizard)
```

Comment:
We always load all needed libraries in the setup chunk. Happy to revert this if you think this reduces readability of the vignette.